tools/debug.sh

```diff
@@ -24,8 +24,13 @@ function main()
     launcher_text_addr=$(get_text_addr "$launcher_path")
     local gdbinit="$script_dir/gdbinit"
 
-    cat >/tmp/x.gdb<<EOF
+    local arch=$(uname -a)
+    if [ "$arch" != "aarch64"]; then
```

Comment:
Please add a space between `"aarch64"` and `]`. From `shellcheck`:

```
    if [ "$arch" != "aarch64"]; then
    ^-- SC1009: The mentioned parser error was in this if expression.
       ^-- SC1073: Couldn't parse this test expression.
                              ^-- SC1020: You need a space before the ].
                              ^-- SC1072: Missing space before ]. Fix any mentioned problems and try again.
```

Reply:
Done